src/main/java/org/apache/commons/imaging/formats/gif/GifImageMetadataItem.java

```diff
@@ -42,13 +44,16 @@ public int getTopPosition() {
         return topPosition;
     }
 
+    public DisposalMethod getDisposalMethod() { return disposalMethod; }
```

Comment:
Just format as other methods and checkstyle test should pass, probably Travis will give the green flag after this :+1:

Reply:
Thanks. Fixed formatting.